simexpal/schemes/schema.json

```diff
@@ -0,0 +1,242 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Validation Schema for Simexpal",
+  "description": "Schema for an experiments.yml used by simexpal",
```

Comment:
Why do we indent with spaces here and not with tabs?

Reply:
This was a default setting for .json files in my code editor, which I overlooked. I changed the indentation to tabs